src/js/disability-benefits/526EZ/helpers.jsx

```diff
@@ -283,3 +296,129 @@ export const evidenceSummaryView = ({ formData }) => {
     </div>
   );
 };
+
+function lowerCaseFirstLetter(word) {
```

Comment:
These helpers from lowerCaseFirstLetter and getPage are actually part of the view field [PR](#7528).